hnn_core/network.py

```diff
@@ -345,6 +376,8 @@ class Spikes(object):
         The inner list contains the type of spike (e.g., evprox1
         or L2_pyramidal) that occured at the corresonding time stamp.
         Each gid corresponds to a type via Network::gid_dict.
+    tstart, tstop : int | float | None
+        Values defining the start and stop times of all trials.
```

Comment:
document tstart and tstop in 2 separate lines just to conform to the conventions